lib/host.py

```diff
@@ -566,3 +566,6 @@ def enable_hsts_header(self):
     def disable_hsts_header(self):
         self.ssh(['rm', '-f', f'{XAPI_CONF_DIR}/00-XCP-ng-tests-enable-hsts-header.conf'])
         self.restart_toolstack(verify=True)
+
+    def get_vdi_sr_uuid(self, vdi_uuid):
```

Comment:
What about moving it to the VDI class?

Reply:
My thought was rather to move it to `Pool` instead.  In `VDI` it would need to be a class- or static-method, and would need to get a ref to the pool anyway.

Reply:
To me, the SR looks like an attribute of the VDI object. Why would it be a static method? You even added the `VDI.param_get` method that you could use.

Reply:
The caller has no VDI object to start with, we want to get one from just a UUID.

Reply:
(moved to `Pool` in last push)

Reply:
I thought we could instanciate objets, but there's a chicken and egg problem, since we ask for a SR object in the constructor. For now, a `Pool` method will do.